app.js

```diff
@@ -1,10 +1,27 @@
+/*jslint node: true */  
```

Comment:
Look at adding a .jshintrc file to drive this type of jshint configuration. See https://raw.githubusercontent.com/asterisk/node-voicemail-data/master/.jshintrc for an example.